docs/user-guide/install-zowe-server-install-wizard.md

```diff
@@ -0,0 +1,226 @@
+* [Choosing the Server Installation Type](#choosing-the-server-installation-type)
+* [Zowe Server Configuration](#zowe-server-configuration)
+* [Final Review](#final-review)
+* [Troubleshooting](#troubleshooting)
+  * [Failure to establish a TLS connection](#failure-to-establish-a-tls-connection)
+  * [Unable to continue with Wizard installation](#unable-to-continue-with-wizard-installation)
+
+
+## Benefits of Wizard installation
+
+- Reduces the need for YAML editing in z/OS Unix by handling these operations for you based on inputs to the prompts presented in the Wizard UI.
+
+- Inputs to the prompts are validated, so that invalid input or typos entered in any commands are prevented.  
+
+- The separation of duties for steps to be performed by a security administrator is made easier, whereby a system programmer or system administrator can skip particular steps where elevated user permissions are required, whereupon security administrators can perform such steps outside of the Wizard.
+
+- Results for each step and the YAML configuration output can be reviewed for reference alongside activites performed outside of the Wizard, or for future use.
+
+## Prerequisites of the Wizard
+
+- Microsoft Windows, Apple Mac OS, or a Linux with an X11 or Wayland server display which can install programs from .rpm or .deb formats 
+- An FTP or FTPS connection to z/OS for Zowe installation
+- An account on z/OS that has access to z/OS UNIX for Zowe installation 
+- A security administrator to configure required permissions in z/OS and z/OSMF
+- A security administrator to generate certificates for Zowe
+- A network administrator to open ports used by Zowe 
+
+## Downloading the Wizard
+
+To download the latest version of the Wizard, visit [Zowe.org](https://www.zowe.org/download.html).
+Ensure that you download the appropriate file extension type according to your operating system:
+
+Operating System | File Extension Type
+---|---
+Microsoft Windows | .exe
+Apple Mac OS | .dmg
+Linux (debian-based) | .deb
+Linux (RedHat or SuSE-based) | .rpm
+
+
+## Installing Zowe server-side components
+
+1. Select **New Zowe Installation**.
+2. In the Connection window, provide details as described in the following table:
+
+### Setup Wizard connection to z/OS
+
+Field name| Description                
+---|---
+Host      |Value for the target z/OS system for Zowe Installation. For example, `mainframe.yourcompany.com`
+FTP Port  |The FTP Port number for internal use. The default port is 21. If not specified, the wizard uses the default port.
+User Name |Your z/OS username.
+Password  |Your z/OS password.
+
+3. Select Use FTP over TLS (Highly Recommended). Provide details as described in the following table:
+        
+Field name | Description
+---|---
+Min TLS    |Select the minimum TLS version to accept the certificate from the server.
+Max TLS    |Select the maximum TLS version to accept the certificate from the server.
+4. Select **Accept all certificates**.
```

Comment:
Wrong. It is not recommended to select "Accept all certificates". Selecting "Accept all certificates" will disable certificate verificiation